src/components/SvgMousePosition/component.js

```diff
@@ -0,0 +1,83 @@
+import { Component } from 'react';
+import { subtract } from 'ramda';
+import PropTypes from 'prop-types';
+import { noop } from 'lodash';
+import { throttle } from 'throttle-debounce';
+import { elementInstancePropTypes } from 'models';
+
+class SvgMousePosition extends Component {
+  state = {
+    mousePosition: {},
+    // eslint-disable-next-line react/no-unused-state
+    svgReact: null,
```

Comment:
Just outta curiosity, why is it needed to add this state as it's not used?

Reply:
I was receiving the eslint error `react/no-unused-state` in my VS Code but actually is not happening when I'm running `yarn lint` 🤔 
I will remove it.